repository_service_tuf_api/api/metadata.py

```diff
@@ -31,6 +31,25 @@ def post(payload: metadata.MetadataPostPayload):
     return metadata.post_metadata(payload)
 
 
+@router.put(
+    "/",
```

Comment:
I think it should be a specific endpoint "/api/v1/metadata/"
Also, I would use the following behavior.
No rolename = bump all online roles.

Reply:
It is already using `api/v1/metadata`:
<img width="1619" alt="image" src="https://github.com/repository-service-tuf/repository-service-tuf-api/assets/16246778/859b8aac-e829-482b-b74e-623867befe2a">

I will address the second comment though.

Reply:
Sorry, what I mean is to use a more specific endpoint `/api/v1/metadata/<choose a name>`
I see the `/api/v1/metadata` for more generic actions like Update Metadata.

As this description says, "Force a new version of online metadata role(s)." it should have a more specific endpoint
i.e.: `/api/v1/metadata/online`. Following our past changes, it is an asynchronous task that could be a POST.

Reply:
After a discussion we agreed it will be `POST /api/v1/metadata/online/`